server/src/routes/api/v1/buildsReviewsRouter.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import { Build, Review } from "../../../models/index.js"
+import objection from "objection"
+const { ValidationError } = objection
+import cleanBuildForm from "../../../services/cleanBuildForm.js";
+
+const buildsReviewsRouter = new express.Router({mergeParams: true})
+
+buildsReviewsRouter.post("/", async (req, res) => {
+    
```

Comment:
no empty line as the first line of a block
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--padded-blocks

Reply:
done